src/lib/protocols/http.c

```diff
@@ -193,10 +193,10 @@ static void ndpi_validate_http_content(struct ndpi_detection_module_struct *ndpi
 
       packet->http_check_content = 1;
 
-      if(len >= 8 /* 4 chars for \r\n\r\n and at least 4 charts for content guess */) {
-	double_ret += 4;
-
-	ndpi_http_check_human_redeable_content(ndpi_struct, flow, double_ret, len);
+      if (len > 4 /* 4 chars for \r\n\r\n and at least 4 charts for content guess */) {
+        double_ret += 4;
+        len -= 4;
+        ndpi_http_check_human_redeable_content(ndpi_struct, flow, double_ret, len);
       }
```

Comment:
Is this change still related to the PE? If so, I think we should update the comment, at very least....

Reply:
Actually not anymore. I guess it can be reverted.